### ISSUE TITLE
How are "observable" satellites determined?

### ISSUE THREAD

**Issue by TheGregor** [osqzss/gps-sdr-sim#183]
It is my understanding that you can input a static location (Lat / Long / Alt) and a start time (hh:mm:ss) and, using an ephemeris file, generate signals for the satellites that should be visible from that location at that time.  However, when I do so I receive data for an entirely different set of satellites than those that were actually observed at the time.  The information is correct, insofar as I can generate a PVT fix that matches the target location, just not using the satellites that were observed in real time.  Any ideas as to what could cause this disparity?  I've verified that the inputs to gps-sdr-sim are correct, and that the correct ephemeris is being used.

**Comment by TheGregor** [comment]
Yes, I was using the -t / -T flag, and between "These files are then used to generate the simulated pseudorange and Doppler for the GPS satellites in view" and  "The simulation start time can be specified if the corresponding set of ephemerides is available.", propagating satellite orbits is exactly what I expected.  Is there any way to do such a thing with this program?  I would prefer to create replicas of specific moments with specific satellites rather than simply spoofing locations.

Thanks!

**Comment by osqzss** [comment]
The ephemeris parameters are only valid for a couple of hours. If you can obtain the latest ephemeris from the live GPS signal, use "-t" (lower case letter) option. It will propagate the satellite orbit to the specified time. I personally recommend to use UBX outputs from a u-blox receiver to obtain the current broadcast ephemeris.

**Comment by chrsengel** [comment]
What is the correct format for the `-T` option?
